pkg/apis/scheduling/v1beta1/types.go

```diff
@@ -261,6 +261,7 @@ type QueueStatus struct {
 type QueueSpec struct {
 	Weight     int32           `json:"weight,omitempty" protobuf:"bytes,1,opt,name=weight"`
 	Capability v1.ResourceList `json:"capability,omitempty" protobuf:"bytes,2,opt,name=capability"`
+	State QueueState `json:"state,omitempty" protobuf:"bytes,1,opt,name=state"`
```

Comment:
That's required to your environment? We already moved this to `QueueStatus` :)

Reply:
cache.go file
```
Spec: vcv1beta1.QueueSpec{
			Reclaimable: &reclaimable,
			State:       "Open",
			Weight:      1,
		},
```
declare default Queue！

Reply:
If this field `QueueStatus` is not needed，Delete the default value in webhook？

Reply:
If this field `State` is not needed，Delete the default value in webhook？